### DISCUSSION TITLE
Missing releases

### DISCUSSION THREAD

**Discussion by GoluLiliput** [brahmkshatriya/echo#3] [Q&A]
Hello,
I can't find any releases of the app.

**Comment by brahmkshatriya** [comment]
Because it isn't ready to be used yet

**Reply by mizu-048** [reply]
alpha?

**Reply by DarkCrypt** [reply]
A self-updating, pre-release version would be sweet.

**Comment by DarkCrypt** [comment]
You can try out the app as it's being built without waiting for an official release.

Download the artifact from the [Actions](https://github.com/brahmkshatriya/echo/actions) tab as they come out. This will be a zip file with the apk inside. You'll also have to install/uninstall every new build since they don't install over each other.

**Reply by brahmkshatriya** [reply]
The [nightly builds](https://github.com/brahmkshatriya/echo/actions/workflows/nightly.yml) are signed, so need to uninstall them,
Alternatively, you can join the discord or the tg channel to get notified when a nightly is released

**Reply by DarkCrypt** [reply]
Nice, I didn't notice the nightly builds. Thank you for the info.

**Comment by GianlucaCise** [comment]
But couldn't they be published as a pre-release?
